CHANGES.md

```diff
@@ -1,4 +1,39 @@
+* Added a `code_version` parameter to the `@graph_asset` decorator [#25219](https://github.com/dagster-io/dagster/pull/25219)
+* If a `LaunchPartitionBackfill` mutation is submitted to GQL with invalid partition keys, it will return an early `PartitionKeysNotFoundError`. [#25107](https://github.com/dagster-io/dagster/pull/25107)
+* `AssetSelection.checks_for_assets` now accepts `AssetKey`s and string asset keys, in addition to `AssetsDefinition`s. [#25161](https://github.com/dagster-io/dagster/pull/25161)
+* [ui] Added a search bar to partitions tab on the asset details page [#25162](https://github.com/dagster-io/dagster/pull/25162)
+* [ui] Restored docked left nav behavior for wide viewports. [#25177](https://github.com/dagster-io/dagster/pull/25177)
+* [dagster-aws] `get_objects` now has a `since_last_modified` that enables only fetching objects modified after a given timestamp. [#25030](https://github.com/dagster-io/dagster/pull/25030)
+* [dagster-aws] New AWS EMR Dagster Pipes client (`dagster_aws.pipes.PipesEMRCLient` ) for running and monitoring AWS EMR jobs from Dagster. [#23998](https://github.com/dagster-io/dagster/pull/23998)
+* [dagster-looker] Pinned the looker-sdk dependency below 24.18.0 to avoid this issue: https://github.com/looker-open-source/sdk-codegen/issues/1518. [#25233](https://github.com/dagster-io/dagster/pull/25233)
+
+### Bugfixes
+
+* Fixed an issue which could cause incorrect evaluation results when using self-dependent partition mappings with `AutomationConditions` that operate over dependencies. [#25278](https://github.com/dagster-io/dagster/pull/25278)
+* [ui] Fixed an issue where the breadcumb on asset pages would flicker nonstop. [#25188](https://github.com/dagster-io/dagster/pull/25188)
+* [dagster-embedded-elt] Fixed extraction of metadata for dlt assets whose source and destination identifiers differ [#25266](https://github.com/dagster-io/dagster/pull/25266)
+* [dagster-databricks] Fixed a permissioning gap that existed with the `DatabricksPySparkStepLauncher`, so that permissions are now set correctly for non-admin users. [#25083](https://github.com/dagster-io/dagster/pull/25083)
+* [dagster-dbt] Fixed an issue where column metadata generated with `fetch_column_metadata` did not work properly for models imported through dbt dependencies [#25185](https://github.com/dagster-io/dagster/pull/25185)
+
+### Documentation
+
+* [dagster-k8s] `DagsterK8sPipesClient.run` now shows up in API docs. [#24678](https://github.com/dagster-io/dagster/pull/24678)
+
+### Dagster Plus
+
+* [ui] Fixed a bug in the catalog UI where owners filters were not applied correctly. [#25142](https://github.com/dagster-io/dagster/pull/25142)
+* [ui] Fixed width of the column lineage dropdown selector on the asset page. [#12024](https://github.com/dagster-io/internal/pull/12024)
```

Comment:
I dont think we want to link internal pull requests?

Reply:
Good point